### PR TITLE
added troubleshooting info about shared drives/files and volume mounting

### PR DESCRIPTION
**Pull request by londoncalling**
### Proposed changes

Added information to several topics about the need for shared drives or file sharing in some cases per user feedback.

### Related issue

#477 

### Please take a look

@ubershmekel, @dnephin , @ManoMarks , @nathanleclaire , @mstanleyjones 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by nathanleclaire** [comment]
Looks great (and long overdue!), my only piece of feedback is that on Windows the mount is usually `C:\Users`, and to be cautious in pieces which are cross-platform (e.g., the Compose getting started) to mention `C:\Users` as well as `/Users`, since there's no `/Users` on Windows.

**Comment by johndmulhausen** [comment]
Deploy preview ready!

Built with commit 53a2316d702f6870c8c0f05565cdee66a974caa1

https://deploy-preview-587--docker-docs.netlify.com

**Comment by ManoMarks** [comment]
LGTM

File: compose/gettingstarted.md [hunk 1]
```diff
@@ -108,7 +108,23 @@ This Compose file defines two services, `web` and `redis`. The web service:
...
+The `redis` service uses the latest public
+[Redis](https://registry.hub.docker.com/_/redis/) image pulled from the Docker
+Hub registry.
+
+>**Tip:** If your project is outside of the `Users` directory, then you need to
+share the drive or location of the Dockerfile and volume you are using. If you
+get runtime errors indicating an application file is not found, a volume mount
+is denied, or a service cannot start, try enabling file or drive sharing. Volume
+mounting requires shared drives for projects that live outside of `C:\Users`
+(Windows) or `/Users` (Linux, Mac), and is required for _any_ project on Docker
```
Comment (dnephin) [inline compose/gettingstarted.md]: This is not for Linux, just for Mac

Comment (londoncalling) [inline compose/gettingstarted.md]: got it, thanks @dnephin

**Comment by nathanleclaire** [comment]
LGTM

**Pull request merged**